src/Meilisearch/TaskEndpoint.cs

```diff
@@ -43,24 +43,12 @@ public async Task<TaskInfo> GetTaskAsync(int taskUid, CancellationToken cancella
         /// <param name="indexUid">Uid of the index.</param>
         /// <param name="cancellationToken">The cancellation token for this call.</param>
         /// <returns>Returns a list of tasks of an index.</returns>
-        public async Task<Result<IEnumerable<TaskInfo>>> GetIndexTasksAsync(string indexUid, CancellationToken cancellationToken = default)
+        public async Task<TasksResults<IEnumerable<TaskRessource>>> GetIndexTasksAsync(string indexUid, CancellationToken cancellationToken = default)
         {
-            return await _http.GetFromJsonAsync<Result<IEnumerable<TaskInfo>>>($"indexes/{indexUid}/tasks", cancellationToken: cancellationToken)
+            return await _http.GetFromJsonAsync<TasksResults<IEnumerable<TaskRessource>>>($"tasks?indexUid={indexUid}", cancellationToken: cancellationToken)
                 .ConfigureAwait(false);
         }
 
-        /// <summary>
-        /// Get one task from an index.
-        /// </summary>
-        /// <param name="indexUid">Uid of the index.</param>
-        /// <param name="taskUid">Uid of the task.</param>
-        /// <param name="cancellationToken">The cancellation token for this call.</param>
-        /// <returns>Return the task of the index.</returns>
-        public async Task<TaskInfo> GetIndexTaskAsync(string indexUid, int taskUid, CancellationToken cancellationToken = default)
```

Comment:
If I'm correct, we agreed to keep those methods but remove the implementation that points to `$"indexes/{indexUid}/tasks/{taskUid}"` and replace it to `$"indexes/{taskUid}"`

Reply:
In the [Issue](https://github.com/meilisearch/integration-guides/issues/205) it says:

> All the changes:

>Remove GET /indexes/:indexUid/tasks. Use GET /tasks?indexUid=:indexUid instead.
Remove GET /indexes/:indexUid/tasks/:taskUid. Use GET /tasks/:taskUid instead.

So I just used the same `GetTaskAsync()` method for client and index